code/index.html

```diff
@@ -1,37 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
-  </head>
-  <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
-
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
-
-    <script src="clue.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Nosifer&display=swap" rel="stylesheet">
+  <title>Cluedo Cards</title>
+</head>
+
+<body>
+
+  <h1>Cluedo Cards</h1>
+
+  <section class="cards">
+    <article class="killer-deck" id="killerCard">
+      <p>KILLER</p>
+      <p id="killerName"></p>
+      <p id="killerAge"></p>
+      <p id="killerOccupation"></p>
+      <img id="killerImage" src="" height="150" />
+      <p id="killerDescription"></p>
+      <p id="favouriteWeapon"></p>
+    </article>
+    <article class="weapon-deck" id="weaponCard">
+      <p>WEAPON</p>
+      <p id="weaponName"></p>
+      <p id="weaponWeight"></p>
+    </article>
+    <article class="room-deck" id="roomCard">
+      <p>ROOM</p>
+      <p id="roomName"></p>
+    </article>
+  </section>
+
+  <h2 id="mystery"></h2>
+
+  <button id="revealButton" class="mystery-button" type="button" onclick>REVEAL THE CRIME</button>
+
+  <script src="clue.js"></script>
+
+</body>
+
+</html>
```

Comment:
Clean code! Even though it might not be necessary in this project, try to always comment on the different sections of your code so that a new person can pick up easily.